modules/support-reviewing-an-access-request-from-the-hybrid-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-the-hybrid-cloud-console_{context}"]
+= Reviewing an access request from the {hybrid-console-second}
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Navigate to link:https://console.redhat.com/openshift[OpenShift Cluster Manager] and select *Clusters*.
+
+. Click the cluster *Name* to review the *Access Request*.
+
+. Select the *Access Requests* tab to list all *states*.
+
+. Select *Open* for the *Pending* state.
+
+. Click *Approve* or *Deny* under *Decision* in the *Access Request Details* page.
```

Comment:
would be "modal" a more accurate term? or is it not intended to be so specific here :thinking: 

```suggestion
. Click *Approve* or *Deny* under *Decision* in the *Access Request Details* modal.
```

Reply:
changed thanks